src/parameterFactory.js

```diff
@@ -0,0 +1,115 @@
+import {
+  DateInput,
+  EmailInput,
+  NullableBooleanInput,
+  NumberInput,
+  TextInput,
+} from 'react-admin';
+import React from 'react';
+
+function guessType(parameter, fields) {
+  if (parameter.variable.match(/.*\[exists\]/i) !== null) {
+    return 'nullableBoolean';
+  }
+
+  if (
+    parameter.variable.match(/.*\[after\]/i) !== null ||
+    parameter.variable.match(/.*\[before\]/i) !== null ||
+    parameter.variable.match(/.*\[strictly_after\]/i) !== null ||
+    parameter.variable.match(/.*\[strictly_after\]/i) !== null
+  ) {
+    return 'date';
+  }
+
+  if (parameter.variable.match(/.*\[between\]/i) !== null) {
+    return 'between';
+  }
+
+  return guessTypeFromResource(parameter, fields);
+}
+
+function guessTypeFromResource(parameter, fields) {
+  const type = fields
+    .filter(field => {
+      return field['name'] === parameter.variable;
```

Comment:
Fixed (parameter.property => parameter.variable) so that range and nullableboolean fields can be generated.

Reply:
Agreed, i renamed from `property` to `variable` in another commit and i forgot this line